tiled/_tests/test_client_cache.py

```diff
@@ -169,3 +173,52 @@ def test_clear_cache(client):
     assert cache.count() > 0
     cache.clear()
     assert cache.size() == cache.count() == 0
+
```

Comment:
See comment regarding `sqlite3.threadsafety` and comparing to a named constant/enum.